documentation/guides/migrating-from-v8-to-v9.md

```diff
@@ -321,6 +341,18 @@ padding: 0;
 list-style: none;
 ```
 
+#### `when-not-printing()`
+
+| Function                      | Replacement Value/Token |
+|------------------------------|-------------------------|
+| `@include when-not-printing` | `@media not print`      |
+
+#### `when-printing()`
+
+| Function                      | Replacement Value/Token |
```

Comment:
```suggestion
| Mixin                      | Replacement Value/Token |
```
Sorry for the total nitpick 😬 just want to keep consistent with the other sections